_about/participating.md

```diff
@@ -93,11 +93,11 @@ and to get information on participation and links to the mailing list
 archives.
 
 -   [ARIA Working
-    Group](https://www.w3.org/WAI/ARIA/ "link to ARIA WG home page")
```

Comment:
Not sure why we have this here and not for other links. Seems superfluous.

Reply:
Agree!